src/GraphBuilder.js

```diff
@@ -49,15 +50,17 @@ export default class GraphBuilder {
    * @param {rdf.Term} baseURI
    * @param {Object} value looks something like this:
    *   { 'resourceTemplate:bf2:Note':
-   *     { '67Bm64T0p2s': { 'http://www.w3.org/2000/01/rdf-schema#label': [Object] },
+   *     { 'rdfClass: 'http://id.loc.gov/ontologies/bibframe/Instance',
+   * '67Bm64T0p2s': { 'http://www.w3.org/2000/01/rdf-schema#label': [Object] },
    *       '555m64M0f3z': { 'http://www.w3.org/2000/01/rdf-schema#label': [Object] } } }
    */
   buildTriplesForNestedObject(baseURI, value) {
     // Is there ever more than one base node?
     for (const key in value) {
       Object.keys(value[key]).forEach((predicateListKey) => {
-        // we don't have a resourceURI to send as the second argument in the propertyTemplates yet.
-        this.buildTriplesForNode(baseURI, null, value[key][predicateListKey])
+        if (predicateListKey !== 'rdfClass') {
```

Comment:
`rdfClass` is being filtered out in 2 locations. Are both necessary? And if both necessary, can filtering be moved into separate method to avoid repeating?

Reply:
I believe they are necessary, because the structure of the redux state is not consistent between the ResourceTemplate and PropertyTemplate levels.

Reply:
See here: https://github.com/LD4P/sinopia_editor/pull/619/files#diff-115c58c1c8d1c22ebd6a017e5b91efe3R78. that key is not present at the ResourceTemplate level.  If they had similar structures, then we could use the same method.

Reply:
@jcoyne and @justinlittman, the current Redux hierarchical structure has been evolving over the project and how best to represent Resource Templates and the relationships with their propertyTemplates including referencing other Resource Templates. Property Templates just represent a triple to the Resource Template and don't have a `rdf:Class`.